snapcraft/models/project.py

```diff
@@ -436,13 +437,36 @@ def _validate_autostart_name(cls, name):
 
         return name
 
-    @pydantic.field_validator("bus_name")
+    @pydantic.field_validator(
+        "command", "stop_command", "post_stop_command", "reload_command", "bus_name"
+    )
     @classmethod
-    def _validate_bus_name(cls, name):
-        if not re.match(r"^[A-Za-z0-9/. _#:$-]*$", name):
-            raise ValueError(f"{name!r} is not a valid bus name")
+    def _validate_command(cls, command: str, info: pydantic.ValidationInfo) -> str:
```

Comment:
`bus-name` is not a command, so maybe it is better to use the `snapd` naming instead of `command`
```suggestion
    def _validate_apps_section_content(cls, command: str, info: pydantic.ValidationInfo) -> str:
```